ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5257,3 +5260,328 @@ kong_apis:
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionHierarchyRemove
+    uris: "{{ collection_prefix }}/v1/hierarchy/remove"
+    upstream_url: "{{ content_service_url }}/collection/v4/hierarchy/remove"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionHierarchyUpdate
+    uris: "{{ collection_prefix }}/v1/hierarchy/update"
+    upstream_url: "{{ content_service_url }}/collection/v4/hierarchy/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionHierarchyRead
+    uris: "{{ collection_prefix }}/v1/hierarchy"
+    upstream_url: "{{ content_service_url }}/collection/v4/hierarchy"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+
+  - name: collectionDialcodeLink
+    uris: "{{ collection_prefix }}/v1/dialcode/link"
+    upstream_url: "{{ content_service_url }}/collection/v4/dialcode/link"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+       - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: collectionCopy
+    uris: "{{ collection_prefix }}/v1/copy"
+    upstream_url: "{{ content_service_url }}/collection/v4/copy"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```

Comment:
ACL has to be contentUpdate/contentAdmin based on the functionmality, need more information on this @rheafernandes